src/google/Scatter.js

```diff
@@ -0,0 +1,215 @@
+
+    Scatter.prototype.publish("xAxisTicks", [], "array", "Replaces The Automatically Generated X-Axis Ticks With The Specified Array",null,{tags:['Private']});
+    Scatter.prototype.publish("yAxisTicks", [], "array", "Replaces The Automatically Generated Y-Axis Ticks With The Specified Array",null,{tags:['Private']});
+
+    Scatter.prototype.publish("xAxisMaxValue", null, "number", "Moves The Max Value of The Horizontal Axis To The Specified Value",null,{tags:['Advanced']});
+    Scatter.prototype.publish("yAxisMaxValue", null, "number", "Moves The Max Value of The Vertical Axis To The Specified Value",null,{tags:['Advanced']});
+
+    Scatter.prototype.publish("xAxisMinValue", null, "number", "Moves The Min Value of The Horizontal Axis To The Specified Value",null,{tags:['Advanced']});
+    Scatter.prototype.publish("yAxisMinValue", null, "number", "Moves The Min Value of The Vertical Axis To The Specified Value",null,{tags:['Advanced']});
+
+    Scatter.prototype.publish("xAxisViewWindowMode", "pretty", "set", "Specifies How To Scale The Horizontal Axis To Render The values Within The Chart Area", ["pretty","maximized","explicit"],{tags:['Advanced']});
+    Scatter.prototype.publish("yAxisViewWindowMode", "pretty", "set", "Specifies How To Scale The Vertical Axis To Render The Values Within The Chart Area", ["pretty","maximized","explicit"],{tags:['Advanced']});
+
+    Scatter.prototype.publish("xAxisViewWindowMax", null, "number", "The Maximum Horizontal Data Value To Render",null,{tags:['Advanced']});
+    Scatter.prototype.publish("yAxisViewWindowMax", null, "number", "The Maximum Vertical Data Value To Render",null,{tags:['Advanced']});
+
+    Scatter.prototype.publish("xAxisViewWindowMin", null, "number", "The Minimum Horizontal Data Value To Render",null,{tags:['Advanced']});
+    Scatter.prototype.publish("yAxisViewWindowMin", null, "number", "The Minimum Vertical Data Value To Render",null,{tags:['Advanced']});
+
+    Scatter.prototype.getChartOptions = function () {
+        var retVal = CommonND.prototype.getChartOptions.apply(this, arguments);
+
+        retVal.aggregationTarget = this.aggregationTarget();
+        retVal.curveType = this.curveType();
+        retVal.pointShape = this.pointShape();
+        retVal.pointSize = this.pointSize();
+        retVal.pointsVisible = this.pointsVisible();
+        retVal.selectionMode = this.selectionMode();
+        retVal.backgroundColor = this.backgroundColor();
+        retVal.dataOpacity = this.dataOpacity();
+        retVal.tooltipIsHtml = this.tooltipIsHtml();
+        retVal.tooltipTrigger = this.tooltipTrigger();
+        
+        retVal.crosshair = {
+            color: this.crosshairColor(),
+            orientation: this.crosshairOrientation(),
+            opacity: this.crosshairOpacity(),
+            trigger: this.crosshairTrigger()
+        };
+
+        retVal.hAxis = {};
+        retVal.vAxis = {};
```

Comment:
I took this block of code straight from another widget.  Since it is used in several different widgets the same way, Matt and I were discussing putting it into a common file in a future commit.  Is it OK if I change it at that point?